pkg/metrics/instance/group_manager.go

```diff
@@ -53,11 +176,12 @@ func (g groupedConfigs) Copy() groupedConfigs {
 
 // NewGroupManager creates a new GroupManager for combining instances of the
 // same "group."
-func NewGroupManager(inner Manager) *GroupManager {
+func NewGroupManager(inner Manager, log log.Logger) *GroupManager {
```

Comment:
nit: can you put log first? We generally put registerers/logs as the first argument.